src/DatabaseGpt.SqlServer/SqlServerDatabaseGptProvider.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Data;
+using System.Text;
+using Dapper;
+using DatabaseGpt.Abstractions;
+using DatabaseGpt.Abstractions.Exceptions;
+using Microsoft.Data.SqlClient;
+
+namespace DatabaseGpt.SqlServer;
+
+public class SqlServerDatabaseGptProvider : IDatabaseGptProvider, IDisposable
+{
+    private readonly SqlServerDatabaseGptProviderConfiguration settings;
+    private bool disposedValue;
+
+    public SqlServerDatabaseGptProvider(SqlServerDatabaseGptProviderConfiguration settings)
+    {
+        this.settings = settings;
+        this.connection = new SqlConnection(settings.ConnectionString);
+        connection.Open();
+    }
+
+    private readonly SqlConnection connection;
+
+    public string Name => "SQL Server";
+    public string Language => "T-SQL";
```

Comment:
Place these properties on top, after the `private` fields. Place a blank line between the properties.